src/Data/List/Lazy.purs

```diff
@@ -326,11 +326,8 @@ findIndex fn = go 0
 findLastIndex :: forall a. (a -> Boolean) -> List a -> Maybe Int
 findLastIndex fn xs = ((length xs - 1) - _) <$> findIndex fn (reverse xs)
 
--- | Insert an element into a list at the specified index, returning a new
--- | list or `Nothing` if the index is out-of-bounds.
--- |
--- | This function differs from the strict equivalent in that out-of-bounds arguments
--- | result in the element being appended at the _end_ of the list.
+-- | Insert an element into a list at the specified index, or append the element
+-- | to the _end_ of the list if the index is out-of-bounds, returning a new list.
```

Comment:
I don't think the italics are necessary here

Reply:
Sure. Was just preserving original text.
Updated PR.

Reply:
Oh oops sorry, I thought that was new. Thanks anyway!